dlp/snippets/deid_test.py

```diff
@@ -291,6 +291,35 @@ def test_deidentify_with_replace_infotype(capsys):
     assert "My favorite site is [URL]" in out
 
 
+def test_deidentify_with_simple_word_list(capsys):
+    word_list = ["RM-GREEN", "RM-YELLOW", "RM_ORANGE"]
+    deid.deidentify_with_simple_word_list(
+        GCLOUD_PROJECT,
+        "My Room is rm-green",
+        "CUSTOM_ROOM_ID",
+        word_list
```

Comment:
If you're using a vertical list of items, add a comma on the last item as well. This is not necessary if they're laid out horizontally, for example: `deid.deidentify_with_simple_word_list(var1, var2, var3)`.
```suggestion
        word_list,
```

Reply:
okay got it, thank you for your comment, Updated that.